commit 8d0ba6b024989da40b7fa982bd074c2d45ecc64b (refs/changes/06/906/3)
Author: Cody Peter Mello <cody.mello@joyent.com>
Date:   2016-11-21T22:30:27+00:00 (2 years, 11 months ago)
    
    MORAY-350 Moray throws ReferenceError: InternalError is not defined
    Reviewed by: David Pacheco <dap@joyent.com>
    Approved by: David Pacheco <dap@joyent.com>

diff --git a/lib/buckets/creat.js b/lib/buckets/creat.js
index b4dca4e..3537109 100644
--- a/lib/buckets/creat.js
+++ b/lib/buckets/creat.js
@@ -15,6 +15,7 @@ var once = require('once');
 var common = require('./common');
 var control = require('../control');
 var BucketConflictError = require('../errors').BucketConflictError;
+var InternalError = require('../errors').InternalError;
 
 
 ///--- Handlers
diff --git a/lib/pg.js b/lib/pg.js
index eb9a693..a85d674 100644
--- a/lib/pg.js
+++ b/lib/pg.js
@@ -25,6 +25,7 @@ var dtrace = require('./dtrace');
 
 var mod_errors = require('./errors');
 var ConnectTimeoutError = mod_errors.ConnectTimeoutError;
+var InternalError = mod_errors.InternalError;
 var InvalidIndexDefinitionError = mod_errors.InvalidIndexDefinitionError;
 var QueryTimeoutError = mod_errors.QueryTimeoutError;
 var UniqueAttributeError = mod_errors.UniqueAttributeError;
